antora-playbook.yml

```diff
@@ -202,6 +202,9 @@ content:
     - url: https://github.com/quarkiverse/quarkus-tika
       branches: [ HEAD, 1.x ]
       start_path: docs
+    - url: https://github.com/quarkiverse/quarkus-web-bundler
+      branches: [ HEAD ]
+      start_path: docs
```

Comment:
Please keep it in alphabetical order. See https://github.com/quarkiverse/quarkiverse-docs/blob/main/antora-playbook.yml#L7

Reply:
Oopsy my alphabet was wrong this morning :) fixing!